dockerfiles/cpubuilder_ubuntu_jammy.Dockerfile

```diff
@@ -15,32 +18,19 @@ RUN apt update && \
         gcc-9 g++-9 \
         ninja-build libssl-dev libxml2-dev libcapstone-dev libtbb-dev \
         libzstd-dev llvm-dev pkg-config
+
+# TODO: re-enable this when LLVM apt packages are working again
 # Recent compiler tools for build configurations like ASan/TSan.
 #   * See https://apt.llvm.org/ for context on the apt commands.
-ARG LLVM_VERSION=19
-RUN echo "deb http://apt.llvm.org/jammy/ llvm-toolchain-jammy-${LLVM_VERSION} main" >> /etc/apt/sources.list && \
-    curl https://apt.llvm.org/llvm-snapshot.gpg.key | gpg --dearmor > /etc/apt/trusted.gpg.d/llvm-snapshot.gpg && \
-    apt update && \
-    apt install -y clang-${LLVM_VERSION} lld-${LLVM_VERSION}
```

Comment:
See https://github.com/llvm/llvm-project/issues/109276

Reply:
Also copied these llvm 19 changes to their own PR for easier review: https://github.com/iree-org/base-docker-images/pull/12

(renaming the file makes dependent PRs tricky though)